frontend/src/components/no_results.svelte

```diff
@@ -29,15 +29,19 @@
         <i>Consider adding more providers or removing all</i>
       </p>
     {/if}
-    {#if !$filters.tvChecked || !$filters.movieChecked}
-      <p class="pt-2">With filter:</p>
+    {#if !$filters.tvChecked || !$filters.movieChecked || !$filters.showNoProviders}
```

Comment:
A dynamic idea

```suggestion
    {#if Object.values($filters).includes(false)}
```

Reply:
We will need to make them all objects with a description for this to work. So either we make a issue and do it in another PR or it should happen here i guess.